src/main/java/io/quarkiverse/spec/generator/deployment/codegen/SpecApiGeneratorStreamCodeGen.java

```diff
@@ -44,20 +44,20 @@ public boolean trigger(CodeGenContext context) throws CodeGenException {
 
         for (final T provider : this.providers) {
             for (BaseSpecInputModel inputModel : provider.read(context)) {
-                LOGGER.debug("Processing OpenAPI spec input model {}", inputModel);
+                LOGGER.debug("Processing spec input model {}", inputModel);
                 if (inputModel == null) {
-                    throw new CodeGenException("SpecInputModel from provider " + provider + " is null");
+                    throw new CodeGenException("Data returned by provider " + provider + " is null");
                 }
                 try {
-                    final Path openApiFilePath = Paths.get(outDir.toString(), inputModel.getFileName());
-                    Files.createDirectories(openApiFilePath.getParent());
+                    final Path specFilePath = Paths.get(outDir.toString(), inputModel.getFileName());
+                    Files.createDirectories(specFilePath.getParent());
                     try (ReadableByteChannel inChannel = Channels.newChannel(inputModel.getInputStream());
-                            FileChannel outChannel = FileChannel.open(openApiFilePath, StandardOpenOption.WRITE,
+                            FileChannel outChannel = FileChannel.open(specFilePath, StandardOpenOption.WRITE,
                                     StandardOpenOption.CREATE)) {
                         outChannel.transferFrom(inChannel, 0, Integer.MAX_VALUE);
-                        LOGGER.debug("Saved OpenAPI spec input model in {}", openApiFilePath);
+                        LOGGER.debug("Saved spec input model in {}", specFilePath);
                         Config config = this.mergeConfig(context, inputModel);
-                        generator.generate(config, openApiFilePath, outDir, getBasePackage(config, openApiFilePath));
+                        generator.generate(config, specFilePath, outDir, getBasePackage(config, specFilePath));
```

Comment:
In this context, the base package should be in the generator context, not here.

Reply:
I was tryinh to reuse the base package resolution, so it is the same for both extensions

Reply:
WDYM by base package resolution? In this context, this property belongs to the inner extension. This class is just passing the property around, not doing anything with it.

Reply:
the property name, the logic is not trivial